deploy/004-omnichain-executor-remote.ts

```diff
@@ -0,0 +1,26 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { LZ_ENDPOINTS, SUPPORTED_NETWORKS } from "../helpers/deploy/constants";
+import { getOmnichainGovernanceExecutorAdminAccount, getSourceChainId } from "../helpers/deploy/deploymentUtils";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const networkName = hre.network.name as SUPPORTED_NETWORKS;
+  const guardian = await getOmnichainGovernanceExecutorAdminAccount(networkName);
+
+  await deploy("OmnichainGovernanceExecutor", {
+    from: deployer,
+    args: [LZ_ENDPOINTS[networkName], guardian, await getSourceChainId(networkName)],
+    log: true,
+    autoMine: true,
+  });
+};
+func.tags = ["OmnichainExecutor", "Remote"];
+
+func.skip = async (hre: HardhatRuntimeEnvironment) =>
+  hre.network.name === "bsctestnet" || hre.network.name === "bscmainnet" || hre.network.name === "hardhat";
```

Comment:
Do we need to skip this on hardhat? For the purpose of testing hardhat could be the remote and source network

Reply:
cb409e8a0fb030b26441249c98c7f018e9a13c3a